.github/workflows/test-e2e.yml

```diff
@@ -7,6 +7,7 @@ on:
         type: boolean
         required: false
         default: false
+  workflow_dispatch:
```

Comment:
@igor-karpukhin we will need to add this line here and in `int-test.yml` so that we can run the tests manually for your PR.